CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 0.33.0-beta.1 (2020-09-12)
+
+This is a beta version to preview the changes in the next release and may not be stable. It is not suggested to install this version for asset management. Welcome any questions or suggestions.
+
+[CKB v0.32.0](https://github.com/nervosnetwork/ckb/releases/tag/v0.32.0) was released on May 22nd, 2020. This version of 
+CKB node is now bundled and preconfigured in Neuron. 
+
+### Refactor
+
+* Upgrade from Electron 7 to Electron 9.
+* Refactor the address storage to be based on a SQLite table storing the public keys for wallets rather than a JSON file.
```

Comment:
=> Refactor the wallet address storage to be based on an SQLite table rather than a JSON file.